fabtests/functional/av_xfer.c

```diff
@@ -109,9 +109,9 @@ static int av_removal_test(void)
 		}
 	}
 
-	fprintf(stdout, "PASS\n");
 	(void) ft_sync();
 out:
+	fprintf(stdout, "%s\n",ret?"FAIL":"PASS");
```

Comment:
nitpick: I would add space when appropriate. same for line 176.

Reply:
Done